Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/1_Behave.xml

```diff
@@ -0,0 +1,32 @@
+<Document>
+# Rule B1: Behave like a normal person.
+You must roleplay. Your character's actions, feelings, and knowledge in-game should be based solely on the character's experiences rather than your own as the player. 
+  - Act in character throughout the round, and focus on collaborating with other players to advance the narrative of the round. Whether or not you do something should be based on whether or not it makes sense for your character to do it.
+  - Let people do their own jobs. Engage with other departments rather than doing everything for yourself. Ask for assistance, and negotiate as needed. 
+  - Actions that have no regard for your character or setting are not acceptable. Roleplaying insanity or any other mental illness as an excuse to bypass this is not allowed. Nanotrasen would not have authorized you to be present on the station. 
+  - Being an antagonist does not allow you to stop playing a character. Determine how your character would react to being given these objectives, and work through it appropriately.
+  - Do not annoy other players without any roleplay or IC reason.
+  - As a non-sentient or partially-sentient character, you are expected to have only a limited understanding of space stations and their mechanics. 
+
+## Rule B1.1: Follow the naming convention rules.
+  - In-character names must not make obvious references nor be a play on words, and must not be obscene. 
+  - Theatrical roles such as the clown, boxer, and mime are allowed stage names with more freedom. 
+  - If you seek to have a name that could trespass onto these rules, talk to an Administrator regarding why this name is important to your character. Administrators reserve the right to approve or deny character names at their discretion.
+
+## Rule B1.2: Communicate in character
+  - Do not use the Emote channel to bypass an inability to speak. Use the Emote channel for things that could reasonably be displayed by your character’s physical motions. 
+  - Do not use Netspeak in character. (lol, brb, wtf, etc)
+  - Do not discuss Out Of Character (OOC) matters In Character (IC). This can include mentioning admins, the fact that one is playing a game, or anything that breaks the immersion of other players. 
+  - Do not use In-Character channels to bypass a lack of ability to use LOOC or OOC chats (e.g. "Huh, wonder where the captain went? OOC He probably left to get dinner lol").
+  - Do not engage in meta-communications (i.e. using external channels to communicate with other players in the same game).
+
+## Rule B1.3: If you die and return to life, follow the new life rules.
+If a player dies, they forget the specific details of their death. [color=#ff0000]Please report players who break this rule.[/color] A borg being removed from its chassis is not a death, and there is no memory loss. 
+  - If they are revived by using a defibrillator, they can only recall vague information such as “Someone shot me” or “I was set ablaze”. 
```

Comment:
It's not clear if the five-minute rule applies here as well. Do you forget the last five minutes but remember vague information? (Gun, fire, cold) If the five-minute rule doesn't apply, then does this mean you are allowed to describe what was around you, say, ten seconds before your death?

I'm not sure this really needs to be clarified as it's more of a rules-lawyery thing to worry about, but it might be worth considering. 🤷

Reply:
The line following this one should answer this question. There is no "5 minutes" anywhere in the defib line for this reason.